README.md

```diff
@@ -289,6 +293,7 @@ Available types:
 - `isVoid` <- whether type is Void (`Void` or `()`)
 - `isTuple` <- whether given type is a tuple
 - `tuple.elements` <- if given type is a tuple returns its elements information (*TupleType.Element*)
+- `attributes` <- type attributes, i.e. `typeName.attributes.escaping`, or `typeName.escaping`
```

Comment:
The reason why I added that is that most of the attributes feel like boolean flags on attributed instances. But yes, skipping `attribute` is not a huge win here.